minidump-stackwalk/get-minidump-instructions.cc

```diff
@@ -78,6 +83,8 @@ void usage()
   fprintf(stderr, "Options:\n");
   fprintf(stderr, "\t--disassemble\tAttempt to disassemble the instructions using objdump\n");
   fprintf(stderr, "\t--address=ADDRESS\tShow instructions at ADDRESS\n");
+ fprintf(stderr, "\t--symbols-url\tA base URL from which URLs to symbol files can be constructed\n");
```

Comment:
I decry reviews that only talk about style, but then find myself pointing out indentation errors...